tests/std/tests/P0645R10_text_formatting_utf8/test.cpp

```diff
@@ -143,8 +139,58 @@ void test_width_estimation() {
         {TYPED_LITERAL(CharT, "\ufe40\u2000\ufe40\x58"), 6},
         {TYPED_LITERAL(CharT, "\ufe40\ufe40\u2000\x58"), 6},
         {TYPED_LITERAL(CharT, "\ufe40\ufe40\ufe40\x58"), 7},
+

```

Comment:
A newline is add here as I find ` {TYPED_LITERAL(CharT, "\u0061\u200D\x58"), 2},` was added in a later pr, and `// Pick "short" and "long" codepoints (\u2000 and \ufe40), then form all permutations of` doesn't apply to them